src/instana/instrumentation/logging.py

```diff
@@ -7,11 +7,11 @@
 import logging
 from collections.abc import Mapping
 
-from ..log import logger
-from ..util.traceutils import get_tracer_tuple, tracing_is_off
+from instana.log import logger
+from instana.util.traceutils import get_tracer_tuple, tracing_is_off
 
 
-@wrapt.patch_function_wrapper('logging', 'Logger._log')
+@wrapt.patch_function_wrapper("logging", "Logger._log")
 def log_with_instana(wrapped, instance, argv, kwargs):
```

Comment:
I am missing type hints in the method signature.

Reply:
Done